Marlin/src/lcd/menu/menu_spindle_laser.cpp

```diff
@@ -50,13 +50,8 @@
 
     if (is_enabled)
       EDIT_ITEM(bool, MSG_CUTTER(STATE), &cutter.state, cutter.disable);
-    else {
-      #if ENABLED(SPINDLE_CHANGE_DIR)
-        EDIT_ITEM(bool, MSG_CUTTER(STATE), &cutter.state, cutter.enable_reverse);
-      #else
-        EDIT_ITEM(bool, MSG_CUTTER(STATE), &cutter.state, cutter.enable_forward);
-      #endif
-    }
```

Comment:
ACTION_ITEM(MSG_CUTTER(ON), cutter.enable_forward);
      #if ENABLED(SPINDLE_CHANGE_DIR)
        ACTION_ITEM(MSG_SPINDLE_REVERSE, cutter.enable_reverse);
      #endif
Looks like my error. Not sure how it got there. But this needs more work.

Reply:
CNC and Laser mixed functions are messy. Not sure what the best path is here.

Reply:
Should these menus be separate?

Reply:
I think the intent of the menu should be to enable forward/reverse mode and a have a separate on/off. That requires a new MSG_SPINDLE_FORWARD string.